lib/next_rails/bundle_report/cli.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+class NextRails::BundleReport::CLI
```

Comment:
Not sure, but there's a possible risk here on apps running the `⁣zeitwerk` gem. 

Sample for zeitwerk conventions I have seeing... If the file name is `client.rb` the class is expected to be called `class Client`, however, this file is named `Cli.rb` and the class all in upper case`class CLI`, I wonder if this is a problem

Reply:
Will check this

Reply:
> I don't know which way to go, but you could add zeitwerk inflections when we need it.

but that involves some steps, I think if the class is `Cli` you don't need to add anything.

> But I also believe that bundle will load the gem and therefore make the constant available - no need for zeitwerk to load it.

true, it can be this way, was just wondering...

> I am confident this works, because I have used [this gem](https://github.com/njh/ruby-mqtt/blob/main/lib/mqtt.rb) in a Rails application and the MQTT constant is available

Awesome, then let's do it 👍